docs/develop/manifest-element-ordering.md

```diff
@@ -0,0 +1,50 @@
+
+The required ordering is specified in the XSD files in the [Schemas](https://github.com/OfficeDev/office-js-docs-pr/tree/docs/overview/schemas) folder. The XSD files are categorized into subfolders for taskpane, content, and mail add-ins.
+
+To find the correct order for the child elements of a given parent element, take the following steps. (This is a simplified process, but XSD files are quite complex. If these steps don't work, there are many resources on the Internet about reading XSD files.)
+
+1. Open the subfolder under [Schemas](https://github.com/OfficeDev/office-js-docs-pr/tree/docs/overview/schemas) for the type of add-in that you are creating. 
+2. Open the XSD file where the parent element is defined as a complex type. (If you don't know which file has the definition, you may have to do step 3 on multiple files till you find it.)
+3. Search for `<xs:complexType name="PARENT_ELEMENT">`, where PARENT_ELEMENT is the name of the parent element.
+4. Inside the definition for the PARENT_ELEMENT, there is (usually) an element called `<xs:sequence>`. The following is the definition for `<SuperTip>` from [TaskPaneAppVersionOverridesV1_0.xsd](https://raw.githubusercontent.com/OfficeDev/office-js-docs-pr/master/docs/overview/schemas/taskpane/TaskPaneAppVersionOverridesV1_0.xsd).
+
+```xml
+  <xs:complexType name="Supertip">
+    <xs:annotation>
+      <xs:documentation>
+        Specifies the super tip for this control.
+      </xs:documentation>
+    </xs:annotation>
+    <xs:sequence>
+      <xs:element name="Title" type="bt:ShortResourceReference" minOccurs="1" maxOccurs="1" />
+      <xs:element name="Description" type="bt:LongResourceReference" minOccurs="1" maxOccurs="1" />
+    </xs:sequence>
+  </xs:complexType>
+```
+
+The `<xs:sequence>` lists the possible child elements, *in the order in which they must appear*. This does *not* necessarily mean that all of them are mandatory. If the `minOccurs` value for a child element is **0**, then the child element is optional. *But if it is present, it must be in the order specified by the `<xs:sequence>` element*.
+
+If there is no `<xs:sequence>` element, or there *is* but the child element is not listed (even though the reference documentation for the child element indicates that it *is* valid for the parent); then the parent element's complex type definition has been extended with additional child elements somewhere else in the XSD file. For example, the definition for the `OfficeApp` complex type does not list `Requirements` as a possible child. But later in the file (within the definition for the `TaskPaneApp` complex type), the definition of `OfficeApp` is extended and `Requirements` is added as an additional valid child.
+
+To find the extended definitions follow these steps:
```

Comment:
Nit: The following steps don't feel like steps, but rather a single step and a note. Maybe that's just my interpretation.